### PR TITLE
English Optimisations Rootdevice

### PR DESCRIPTION
**Pull request by kevdagoat**

### CODE REVIEW

**Comment by kevdagoat** [comment]
There is a few comments I have added, please look over them.

File: decompressed/base/etc/init.d/rootdevice [hunk 1]
```diff
@@ -547,7 +546,7 @@ transformer_lib_check() {
 		logger_command "Found gui here: "$gui_pos
 		if [ $gui_pos != "" ] && [ -s $gui_pos ]; then
 			bzcat $gui_pos | tar -C / -xf - usr #reapply the upgrade as in the gui we store some of this file that we restored
-			logger_command "Restoring transformer lib"
+			logger_command "Restoring transformer lib" #What is going on here? Doesn't even restart transformer???
```
Comment (kevdagoat) [inline decompressed/base/etc/init.d/rootdevice]: What is going on?? Doesn't restart transformer at all

File: decompressed/base/etc/init.d/rootdevice [hunk 2]
```diff
@@ -122,7 +124,6 @@ setup_network() {
 	if [ ! "$(uci -q get network.lan.pppoerelay)" ]; then
 		uci -q add_list network.lan.pppoerelay=''
 	fi
-	sed -i -e 's/option pppoerelay/list pppoerelay/g' /etc/config/network
```
Comment (kevdagoat) [inline decompressed/base/etc/init.d/rootdevice]: Whaat???? I didn't delete this?!

File: decompressed/base/etc/init.d/rootdevice (hunk 1 above)
Comment (kevdagoat) [reply decompressed/base/etc/init.d/rootdevice]: Oh im blind

Comment (kevdagoat) [reply decompressed/base/etc/init.d/rootdevice]: Looks like it doesn't do anything

File: decompressed/base/etc/init.d/rootdevice (hunk 2 above)
Comment (FrancYescO) [reply decompressed/base/etc/init.d/rootdevice]: probably you started editing before https://github.com/Ansuel/tch-nginx-gui/commit/a67c252ff098218529fa5cb8e0b654ea625013bc

File: decompressed/base/etc/init.d/rootdevice [hunk 3]
```diff
@@ -1464,7 +1487,7 @@ ledfw_rework_TG799() {
 
 ledfw_rework_TG800() {
 	if [ ! "$(uci get -q button.info)" ]; then
-		logger_command "Extracting ledfw_support-specificTG800.bz2 and setting up wifi button..."
+		logger_command "Extracting ledfw_support-specificTG800.bz2 and setting up status (wifi) button..."
```
Comment (FrancYescO) [inline decompressed/base/etc/init.d/rootdevice]: huh? was not wps button?

Comment (kevdagoat) [reply decompressed/base/etc/init.d/rootdevice]: ???

Comment (kevdagoat) [reply decompressed/base/etc/init.d/rootdevice]: Says WiFi

File: decompressed/base/etc/init.d/rootdevice (hunk 2 above)
Comment (kevdagoat) [reply decompressed/base/etc/init.d/rootdevice]: Ok

File: decompressed/base/etc/init.d/rootdevice (hunk 3 above)
Comment (FrancYescO) [reply decompressed/base/etc/init.d/rootdevice]: idk i don't remember finally how we configured it, but i remember was the wps button and not wifi

**Comment by kevdagoat** [comment]
Ok then

**Pull request merged**